README.md

```diff
@@ -5,58 +5,47 @@
 [![GitHub stars](https://img.shields.io/github/stars/infinum/eightshift-boilerplate.svg?style=for-the-badge&label=Stars)](https://github.com/infinum/eightshift-boilerplate/)
 [![license](https://img.shields.io/github/license/infinum/eightshift-boilerplate.svg?style=for-the-badge)](https://github.com/infinum/eightshift-boilerplate)
 
-# Eightshift WordPress Boilerplate
+## About Eightshift Boilerplate
 
-This repository contains all the tools you need to start building a modern WordPress project, using all the latest front end development tools.
+Eightshift Boilerplate is a modern WordPress theme boilerplate. It includes everything you need to start building a modern WordPress project, using modern front end development tools, and it's at the core of the Eightshift Development Kit. [Find out more in Eightshift Docs &rarr;](https://infinum.github.io/eightshift-docs/)
 
-## :books: Table of contents
-- [:mortar_board: Documentation](#documentation)
-- [:tada: Awesome features](#awesome-features)
-- [:school_satchel: Requirements](#requirements)
-- [:rocket: Quick start](#quick-start)
-- [:mailbox: Who do I talk to?](#who-do-i-talk-to)
-- [:scroll: License](#license)
+## ⚠️ Requirements
+To get started, you'll need:
 
-## Documentation
-**For more documentation details follow [this link](https://infinum.github.io/eightshift-docs).**
+* [npm](https://www.npmjs.com/)
+* [composer](https://getcomposer.org/)
+* [git](https://git-scm.com/)
+* [WP-CLI](https://wp-cli.org/)
 
-## Awesome features
+## 🏁 Quick start
 
-Here are some buzzwords why you should use Eightshift Boilerplate for your next project. For more documentation details follow [this link](https://infinum.github.io/eightshift-docs).
+If you'd like to get started ASAP, run `npx create-wp-project` inside of `wp-content/themes` to bootstrap your Eightshift Development Kit project.
 
-- Custom Block Editor blocks
-- OOP
-- WP-CLI
-- Webpack
-- SASS
-- Autoprefixer
-- JS Compiler (Babel)
-- Synchronised browser testing
-- Minifying
-- PHPCS
-- Easy SASS media queries
-- Cache busting
-- Eightshift libs
-- Eightshift Frontend Libs
+[Documentation on getting started with Eightshift Boilerplate is available in Eightshift Docs &rarr;](https://infinum.github.io/eightshift-docs/docs/theme/)
 
-## Requirements
+## 📚 Documentation
 
-You can find all requirements on [this link](https://infinum.github.io/eightshift-docs/docs/requirements).
+Eightshift Development Kit comes with [comprehensive documentation](https://infinum.github.io/eightshift-docs).
```

Comment:
```suggestion
Eightshift Development Kit comes with a [comprehensive documentation](https://infinum.github.io/eightshift-docs).
```

Reply:
"Documentation" is very rarely used as a countable noun (see https://en.wiktionary.org/wiki/documentation#Etymology_1), and we don't use it as a countable noun anywhere on the site?